### PR TITLE
Add Shiori extension

### PR DESCRIPTION
**Pull request by namiwang**
## Description

This PR adds an extension Shiori to interact with [Shiori](https://github.com/go-shiori/shiori/) the bookmark manager. 

## Screencast

Screenshot 2024-02-03 at 12 27 50

Screenshot 2024-02-03 at 12 24 45

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/shiori/src/addBookmark.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        />
+      }
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm onSubmit={handleSubmit} />
+        </ActionPanel>
+      }
+    >
+      {/* TODO validation */}
+      <Form.TextField id="url" title="URL to bookmark" />
```
Comment (pernielsentikaer) [inline extensions/shiori/src/addBookmark.tsx:161]:
```suggestion
      <Form.TextField id="url" title="URL to Bookmark" />
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- You can use [useForm](https://developers.raycast.com/utilities/react-hooks/useform) for states, errors and validation

- Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension using the built-in tool instead

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by namiwang** [comment]
@pernielsentikaer thanks for the review! I've addressed the issues and please help review the changes. Thanks!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**